src/components/Products/Products.component.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Pagination from "../Pagination/Pagination.component";
+// import products from "../../mocks/products.json";
+
+function Products(params) {
+  const { results } = params.data
+  const categories = params.categories || []
+
+  return (
+    <div className="featured">
+      <h2>{params.header}</h2>
+      <div className="featured-products">
+        {results
+          .filter((item) => {
+              if (categories.length === 0) {
+                return true
+              } else {
+                return categories.includes(item.data.category.slug)
+              }
+              })
+          .map((product) => (
+            <div className="featured-product" key={product.id}>
```

Comment:
this whole section of what is inside the map can be converted to an individual component ie `FeaturedProduct` that receives its id, url, name, price
```suggestion
            export const FeaturedProduct = ( id, url, alt, name, price ) => {} .... 
```

Reply:
i realized that i wrote my suggestion in an arrow function but there is not any difference if you do it like this o like a normal function

Reply:
Thanks for the advice! I made the component for that :D